.env.example

```diff
@@ -1,18 +1,10 @@
 PORT=
```

Comment:
This env example should get updated with correct variables please

Reply:
Done